src/planscape/plan/views.py

```diff
@@ -190,3 +199,46 @@ def create_project(request: HttpRequest) -> HttpResponse:
         return HttpResponse(str(project.pk))
     except Exception as e:
         return HttpResponseBadRequest("Ill-formed request: " + str(e))
+
+
+def get_scores(request: HttpRequest) -> HttpResponse:
+    try:
+        plan = get_plan_by_id(request.GET)[0]
+        geo = plan.geometry
+        reg = plan.region_name.removeprefix('RegionName.').lower()
+
+        accumulator = RasterPixelAccumulator(geo)
+
+        ids_to_conditions = {
+            c.id: c.condition_name
+            for c in BaseCondition.objects.filter(region_name=reg).all()}
+        ids_to_raster_names = {
+            c.condition_dataset_id: c.raster_name
+            for c in Condition.objects.filter(is_raw=False).all()}
+
+        for id in ids_to_raster_names.keys():
+            if id not in ids_to_conditions.keys():
+                continue
+            for r in ConditionRaster.objects.filter(
+                    name=ids_to_raster_names[id]).all():
+
+                if r.raster is None:
+                    continue
+                accumulator.process_raster(
+                    r.raster, ids_to_conditions[id])
+
+        conditions = []
+        for c in accumulator.stats.keys():
+            if accumulator.stats[c]['count'] == 0:
```

Comment:
If count = 0, I can either add a condition without mean_score to the response or leave it out completely.
Would an empty condition be useful?

Reply:
resolving.